pkg/cassandra/session.go

```
@@ -108,6 +108,31 @@ func (s *Session) Select(ctx context.Context, query string, values ...interface{
 	return rows, nil
 }
 
+// scanRows drains an iterator into rows grouped by the value of the first
+// selected column, which is what tells timeseries apart and therefore has to
+// be convertible to a string.
+func scanRows(iter *gocql.Iter) (map[string][]Row, error) {
```

golangci-lint (2.12.2)

[error] func scanRows is unused

(unused)